src/test/parse-fail/lex-bad-char-literals.rs

```diff
@@ -25,8 +25,7 @@ static s: &'static str =
     "\●" //~ ERROR: unknown character escape
 ;
 
-// THIS MUST BE LAST, since it kills the lexer
-
-static c: char =
-    '●  //~ ERROR: character literal may only contain one codepoint
+static invalid_c: char =
+    'nope' //~ ERROR: character literal may only contain one codepoint: 'nope'
```

Comment:
I’d like more test cases (and would prefer you didn’t remove the current one). It is fine to create more files to contain them:

* `'č`
* `'nope'`
* `'\x00\x00'`
* `'●●'`
* etc